test/http.test.ts

```diff
@@ -0,0 +1,36 @@
+import { HttpCode, HttpError } from "../src/common/http"
+
+describe("http", () => {
+  describe("HttpCode", () => {
+    it("should return the correct HTTP codes", () => {
+      expect(HttpCode.Ok).toBe(200)
+      expect(HttpCode.Redirect).toBe(302)
+      expect(HttpCode.NotFound).toBe(404)
+      expect(HttpCode.BadRequest).toBe(400)
+      expect(HttpCode.BadRequest).toBe(400)
```

Comment:
`BadRequest` appears twice!

Reply:
Definitely a typo - thanks for catching that!